### ISSUE TITLE
Developer Guide Review

### ISSUE THREAD

**Issue by ChoonSiang** [AY2324S2-CS2113-T13-2/tp#103]
- [ ] Missing ":" in the sequence diagram
![image](https://github.com/AY2324S2-CS2113-T13-2/tp/assets/36498233/4c8db3a2-7303-4f37-9d17-264548dbb903)

- [ ] Activation bar being cut off?
![image](https://github.com/AY2324S2-CS2113-T13-2/tp/assets/36498233/ceddcb2a-0068-46c0-8951-c79dad0cfc3b)

- [ ] Use of non standard UML notation
- Accessibility should be indicated using +/- instead of green or red shapes
- "C" should not be there to indicate a class
- Please look up on how to turn off those notation in PlantUML
![image](https://github.com/AY2324S2-CS2113-T13-2/tp/assets/36498233/9aee4a5f-2ace-414d-9a13-852453b286ce)

- [ ] Class level method should not be called this way, refer to the screenshot below to see how it should be indicated.
![image](https://github.com/AY2324S2-CS2113-T13-2/tp/assets/36498233/5feca999-465a-49c4-a2ea-188722d2b914)
![image](https://github.com/AY2324S2-CS2113-T13-2/tp/assets/36498233/3a20a62c-1ec3-4d99-a5d4-4869fee25f96)

- [ ] Why is there a System? Is this part of your program?
![image](https://github.com/AY2324S2-CS2113-T13-2/tp/assets/36498233/9bf01523-16eb-423b-b5c4-49f7d31edd6e)

- [ ] Should there be a << >> when indicating use.
![image](https://github.com/AY2324S2-CS2113-T13-2/tp/assets/36498233/fd300c42-850f-43ec-b428-b6226ff74928)

- [ ] Try to include Target user profile, Value proposition, Non-Functional Requirements, Glossary and Instructions for manual testing

Overall good use of UML diagrams.
